rewrite-maven/src/main/java/org/openrewrite/maven/internal/MavenPomDownloader.java

```diff
@@ -60,6 +60,7 @@ public class MavenPomDownloader {
     private static final RetryConfig retryConfig = RetryConfig.custom()
             .retryOnException(throwable -> throwable instanceof SocketTimeoutException ||
                                            throwable instanceof TimeoutException)
+            .maxAttempts(5) // what should we set?
```

Comment:
What value should I set? I picked `5` arbitrarily, Maybe we can make this configurable with a system property?